brightwind/demo_datasets/solar_iea43_wra_data_model_v1_0.json

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
@olivia-bentley note that I have updated this solar json file on #453 branch to be sure to have some key info needed for testing, please update here to use same file as for that issue